pkg/log/ansHook.go

```diff
@@ -0,0 +1,109 @@
+package log
```

Comment:
Not really, as it is a logrus hook implementation. We can move it to the ans package, but then we do not get the "cyclic import" compilation error. Which is problematic because we should not use the Piper log implementation in the call to `Fire` as we would end up looping indefinitely:
1. A log entry calls `Fire`
2. `Fire` creates a log entry
3. Repeat